adafruit_featherwing/joy_featherwing.py

```diff
@@ -0,0 +1,284 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2018 Kattni Rembor for Adafruit Industries LLC
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+"""
+`adafruit_featherwing.joy_featherwing`
+====================================================
+
+Helper for using the `Joy FeatherWing <https://www.adafruit.com/product/3632>`_.
+
+* Author(s): Kattni Rembor
+"""
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_FeatherWing.git"
+
+import adafruit_seesaw
+from adafruit_featherwing import shared
+from micropython import const
+
+BUTTON_A = const(1 << 6)
+BUTTON_B = const(1 << 7)
+BUTTON_Y = const(1 << 9)
+BUTTON_X = const(1 << 10)
+BUTTON_SELECT = const(1 << 14)
+# BUTTON_MASK = const((1 << BUTTON_A) | (1 << BUTTON_B) | (1 << BUTTON_Y) | (1 << BUTTON_X) |
+#                      (1 << BUTTON_SELECT))
```

Comment:
This should be deleted. You took me too literally on how long it should live. :-)